### ISSUE TITLE
can not import this lib in CRA 5.0 project

### ISSUE THREAD

**Issue by qwaszx102938** [repetere/jsonx#721]
why cant i use this lib in CRA 5.0 (create react app).THE exception is jsonx . there is no export on jsonx

**Comment by yawetse** [comment]
Closing as part of repository maintenance after #1088 landed on main with current dependency updates and passing tests/build. Please open a new issue with a minimal reproduction if this still reproduces on current main or the next published package.
